recipes/perl-test-base/meta.yaml

```diff
@@ -8,25 +8,23 @@ source:
   md5: ced0cd86b099f9fd2cd73df1f46ab5e9
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
-    - perl-threaded
+    - perl
     - perl-spiffy
     - perl-text-diff
     - perl-test-deep
 
   run:
-    - perl-threaded
+    - perl
     - perl-spiffy
     - perl-text-diff
 
 test:
   imports:
     - Test::Base::Filter
-  # Perl 'use' tests trigger "readline() on unopened filehandle DATA" errors,
-  # so we have to use this "dummy" test package instead.
```

Comment:
I suppose this is a useful comment as it explains the reason for the "non-standard" test case. I'd vote to keep it as long as that workaround is needed.

Reply:
good catch! I am adding it back.